README.md

```diff
@@ -1 +1,25 @@
-# hw01
\ No newline at end of file
+#*HW01 - Adam Berger *
+
+##Biography
```

Comment:
You need to add a space between # and content, otherwise the headings will not show.